modules/ROOT/pages/api-groups-deprecating-groups.adoc

```diff
@@ -0,0 +1,24 @@
+= Deprecate API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You might want to deprecate an API Group if a more recent version of the group exists. Deprecating an API Group from API Manager does not deprecate the API Group asset in Exchange. 
```

Comment:
same, the group cannot be deprecated. The group instance can be deprecated. If it is deprecated, it won't accept new contracts.

Reply:
Done! Thanks!